source/connect.txt

```diff
@@ -38,204 +37,200 @@ This is equivalent to the following command:
+To specify a remote host and port, you can use either:
 
-  .. example:: 
+- A :manual:`connection string </reference/connection-string>` with the
+  chosen host and port.
 
-     To connect to a MongoDB
-     instance running on a remote host on port 28015:
+- The ``--host`` and ``--port`` command-line options. If you omit the
+  ``--port`` option, ``mongosh`` uses the default port 27017.
 
-     .. code-block:: sh
+For example, the following commands connect to a MongoDB deployment
+running on host ``mongodb0.example.com`` and port 28015:
 
-        mongosh "mongodb://mongodb0.example.com:28015"
+.. code-block:: sh
 
-  .. note:: Connecting to |service|
-  
-     If your remote host is a |service-fullname| cluster, you can copy 
-     your connection string from the |service| UI. To learn more, see 
-     :atlas:`Connect to a Cluster </connect-to-cluster/#use-the-connect-dialog-to-connect-to-your-cluster>`.
+   mongosh "mongodb://mongodb0.example.com:28015"
 
-- The command-line options ``--host`` and ``--port``. If you do not 
-  include the ``--port`` option, ``mongosh`` uses the **default port** 
-  27017.
+.. code-block:: sh
 
-  .. example:: 
+   mongosh --host mongodb0.example.com --port 28015
 
-     To connect to a MongoDB
-     instance running on a remote host on port 28015:
+.. note:: Connect to |service-fullname|
 
-     .. code-block:: sh
+   If your remote host is an |service| cluster, you can copy your
+   connection string from the |service| UI. To learn more, see
+   :atlas:`Connect to a Cluster
+   </connect-to-cluster/#use-the-connect-dialog-to-connect-to-your-cluster>`
+   in the Atlas documentation.
 
-        mongosh --host mongodb0.example.com --port 28015
+Specify Connection Options
+--------------------------
 
-Connection Options
-------------------
+Specify different connection options to connect to different types of
+deployments.
 
-Connect with Authentication
+Connect With Authentication
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To connect to a MongoDB instance requires authentication, use the 
-``--username`` and ``--authenticationDatabase`` command-line options. 
-``mongosh`` prompts you for a password, which it masks as you type.
-  
-.. example:: 
-
-   To connect to a remote MongoDB instance and authenticate against 
-   the ``admin`` database as user ``alice``:
-
-   .. code-block:: sh
+To connect to a MongoDB deployment that requires authentication, use the
+:option:`--username <--username>` and :option:`--authenticationDatabase
+<--authenticationDatabase>` options. ``mongosh`` prompts you for a
+password, which it hides as you type.
 
-      mongosh "mongodb://mongodb0.example.com:28015" --username alice --authenticationDatabase admin
+For example, to authenticate on the ``admin`` database as user
+``alice``, run the following command:
 
-.. note::
+.. code-block:: sh
 
-   To provide a password with the command instead of using the masked 
-   prompt, you can use the ``--password`` option.
```

Comment:
We don’t want to tell users about `--password` anymore?

Reply:
I wasn't sure about this. I removed it because users are still prompted to specify the password when they omit the ``--password`` option, and not using the ``--password`` option ensures that the password is never viewable as plaintext on the user's screen. I can add this note back in if we still want users to be aware of this slightly less secure method of specifying the password?

Reply:
Yeah, I’ll leave this up to you. The idea isn’t really that human users, which I assume this guide here is targeting, would use it, and more that it’s a useful flag to provide for programmatic usage of mongosh.

Reply:
Ah, that's a good point about programmatic usage. I'll add the note back in, because it's likely good for drivers users to be aware of.